lib/cog/relay/relays.ex

```diff
@@ -130,7 +131,7 @@ defmodule Cog.Relay.Relays do
     {success_bundles, failed_bundles} = announcement
     |> Map.fetch!("bundles")
     |> Enum.map(&lookup_or_install/1)
-    |> Enum.partition(&Util.is_ok?/1)
+    |> Enum.partition(&Helpers.is_ok?/1)
```

Comment:
What's the motivation here? `Util` and `Helpers` intuitively feel like equivalent names.